scripts/feature_tests/upgrade/upgrade.sh

```diff
@@ -9,4 +9,4 @@ source "upgrade_vars.sh"
 
 export ACTION="upgrading"
 
-"${METAL3_DIR}"/scripts/run.sh
+"${METAL3_DIR}"/scripts/run.sh || true
```

Comment:
Will not this mean that it will just continue even if upgrade fails and we may not notice it?

Reply:
Ouch!!! That was used for debugging. I forgot to remove it. Good catch, @lentzi90